app/markets/MarketDetails.tsx

```diff
@@ -90,7 +90,7 @@ export const MarketDetails = ({ id }: MarketDetailsProps) => {
           </div>
           {tab === 'bet' && (
             <div className="p-2">
-              {!marketModel.isClosed ? (
+              {!marketModel.isClosed && marketModel.hasLiquidity ? (
```

Comment:
should we show a diff message like "market has no liquidity" ?

I think that might make sense